src/Collectors/VersionedCollector.php

```diff
@@ -161,10 +161,11 @@ protected function getRecordsForDeletion(array $classes): array
     {
         $keepLimit = (int) $this->config()->get('keep_limit');
         $recordLimit = (int) $this->config()->get('deletion_record_limit');
-        $keepUnpublishedDrafts = (bool) $this->config()->get('keep_unpublished_drafts');
```

Comment:
This line should stay

Reply:
Hmm not sure what happened there, must have accidently deleted it. Will add it back.